src/panels/NightSkyPanel/tabs/NightSkyMarkingsTab.tsx

```diff
@@ -0,0 +1,147 @@
+import { Button, Grid, Text } from '@mantine/core';
+
+import { useOpenSpaceApi } from '@/api/hooks';
+
+import { IconLabelButton } from '../components/IconLabelButton';
+
+export function NightSkyMarkingsTab() {
```

Comment:
This component needs to behave well when the actions or object are not included in the scene